org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/converter/ZigBeeBaseChannelConverter.java

```diff
@@ -318,7 +319,7 @@ protected ChannelUID createChannelUID(ThingUID thingUID, ZigBeeEndpoint endpoint
      * @param level an integer between 0 and 254
      * @return the scaled {@link PercentType}
      */
-    protected PercentType levelToPercent(int level) {
+    public PercentType levelToPercent(int level) {
```

Comment:
To be honest I was a bit too lazy - these operations are used by the ```ZigBeeBaseChannelConverterTest```. I will move the test to its corresponding package so that they can be accessed again using the protectected access level

Reply:
Personally I'm not so keen to raise the visibility just for test. Were you planning on changing these back to private if they don't need to be public?

Reply:
Sry I had already reduced the visibility ... must be a mistake happened probably while fixing merge conflicts ... will reduce the visibility again

Reply:
Just checked .... the updated PR contains already the reduced the visibility ...  dont know why this is not displayed here as outdated ...

are there any additional remarks?